getting-started/chat-thread.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Chat and thread"
+weight: 10
+---
+
+# Chat and thread
+
+
+### Basics
+
+In our system each chat is divided into threads. Threads are standalone parts of a chat and can contain events. Multiple users (customers or agents) can participate in a chat and every user can have multiple chats at the same time.
+
+New threads within a chat are created on the server side, algorithm of this is discussed below in [Advanced](#advanced) section.
```

Comment:
algorithm of this -> the algorithm of this

in [Advanced](#advanced) section. -> in the [Advanced](#advanced) section.